### ISSUE TITLE
clarify (lack of) need for added parser for calc tests

### ISSUE THREAD

**Issue by bremner** [shriram/plai-public#30]
### Contact Details

bremner@unb.ca

### Which part are you commenting on?

p. 133

### What's your suggestion?

You write "with a suitable parser", but this is confusing (for me) because the tests run as is.

**Comment by shriram** [comment]
Not necessarily. If their parser is still the one from page 34, I don't think they would have string concatenation or even string constants, I believe?

We haven't really kept tabs on parsing since then — I've been assuming it has been extended all along, implicitly — and I can't guarantee what state their parser is in; hence the fudge-phrase.

But I want to make that more explicit in the next version so that we can avoid this ambiguity. So I'm going to ignore this for now but only because it should be taken care of *by design* in the next version.
